matcher_test.go

```diff
@@ -0,0 +1,36 @@
+package errors
+
+import (
+	"io"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIs(t *testing.T) {
+	k := New("foo")
+	err := k.New("bar")
+
+	assert.Equal(t, Is(err, k), true)
+	assert.Equal(t, Is(io.EOF, k), false)
+}
+
+func TestIsMultiple(t *testing.T) {
```

Comment:
test with no matchers returning false (or true depending on what we want) would be nice